src/Test/Utilities/Desktop/TestResource.resx

```diff
@@ -123,7 +123,7 @@
 #pragma warning disable 414, 3021
```

Comment:
So, should I change file extension for getting rid of any errors? Or just enough to extract this code from .resx to separate .cs file?